sdk/cosmos/azure-cosmos/test/aggregate_tests.py

```diff
@@ -96,8 +96,8 @@ def _setup():
             _insert_doc(mcs.created_collection, document_definitions)
 
         def _generate_test_configs():
-            aggregate_query_format = 'SELECT VALUE {}(r.{}) FROM r WHERE {}'
-            aggregate_orderby_query_format = 'SELECT VALUE {}(r.{}) FROM r WHERE {} ORDER BY r.{}'
+            aggregate_query_format = 'SELECT TOP 1 VALUE {}(r.{}) FROM r WHERE {}'
```

Comment:
Why do we need to add TOP here?

Reply:
Fixed a bug in the query pipeline where top was being called ahead of aggregates. Results were incorrect. Added TOP here instead of writing new TOP + Aggregate queries